feedthefox/devices/templates/jinja2/devices.html

```diff
@@ -9,32 +9,32 @@
 
     <div class="row">
       <div class="col-sm-3">
-        <form>
+        <form action="" method="GET">
           <div class="form-group">
             <label>Manufacturer</label>
-            <select class="form-control" name="manufacturer">
-              <option value="0" selected>All</option>
-              <option value=""></option>
-            </select>
+            {{ field_with_attrs(device_filter.form.manufacturer, class="form-control") }}
           </div>
           <div class="form-group">
             <label>Type</label>
-            <select class="form-control" name="type">
-              <option value="0" selected>Smartphone</option>
-            </select>
+            {{ field_with_attrs(device_filter.form.type, class="form-control") }}
```

Comment:
The idea is that we will have different types at some point, so why not facilitate it in a more generic way?